### PR TITLE
Poincare Ball model log/exp  fix

### PR DESCRIPTION
**Pull request by tgeral68**
- fix exp/log Poincare Ball
- backend adding arctanh
- backend docstring

### CODE REVIEW

File: geomstats/backend/pytorch/__init__.py [hunk 1]
```diff
@@ -320,6 +320,8 @@ def trace(*args, **kwargs):
 def mod(*args, **kwargs):
     return torch.fmod(*args, **kwargs)
 
+def arctanh(x):
```
Comment (ninamiolane) [inline geomstats/backend/pytorch/__init__.py]: wow, I didn't know pytorch doesn't have this: https://github.com/pytorch/pytorch/issues/10324.
thanks!

File: tests/test_backend_pytorch.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 """
-Unit tests for numpy backend.
+Unit tests for pytorch backend.
```
Comment (ninamiolane) [inline tests/test_backend_pytorch.py]: Thank you

File: tests/test_hyperbolic_space_coordinates.py [hunk 3]
```diff
@@ -113,10 +113,9 @@ def test_distance_ball_extrinsic_from_extr_5_dim(self):
         dst_extr = extrinsic_metric.dist(x_extr, y_extr)
         self.assertAllClose(dst_ball, dst_extr)
 
-    @geomstats.tests.np_only
     def test_log_exp_ball_extrinsic_from_extr(self):
-        x_int = gs.array([[10., 0.2]])
-        y_int = gs.array([[5., 3]])
+        x_int = gs.array([[1., 0.2]])
```
Comment (ninamiolane) [inline tests/test_hyperbolic_space_coordinates.py]: Argh, numerical problems?

**Comment by ninamiolane** [comment]
Yann, Yann, Nico, Stefan, Christian, Olivier and I say hi ;) Safe travels!

File: tests/test_hyperbolic_space_coordinates.py (hunk 3 above)
Comment (tgeral68) [reply tests/test_hyperbolic_space_coordinates.py]: yes in pytorch only

Comment (tgeral68) [reply tests/test_hyperbolic_space_coordinates.py]: Maybe change pytorch default type ?

**Comment by ninamiolane** [comment]
Good to go as soon as the flake8 is solved!

File: tests/test_hyperbolic_space_coordinates.py (hunk 3 above)
Comment (ninamiolane) [reply tests/test_hyperbolic_space_coordinates.py]: from float32 to float 64 you mean?

**Pull request merged**